src/client/components/detail-chart/detail-chart.stories.js

```diff
@@ -36,12 +36,18 @@ const unit = {
   water: 'cm',
 };
 
+const { sensors_readings } = data;
```

Comment:
maybe sensorsReadings instead
```suggestion
const { sensorsReadings } = data;
```